rca/project_styleguide/templates/patterns/organisms/news/news_api_content.html

```diff
@@ -0,0 +1,27 @@
+<div class="news">
+    {% for item in news %}
+
+        {% if forloop.first %}
+            <div class="news__feature">
+        {% endif %}
+
+        {% if forloop.counter == 2 %}
+            <div class="news__sub-features">
+        {% endif %}
+
+        {% include "patterns/molecules/card/card_api_content.html" with counter=forloop.counter %}
+
+        {% if forloop.first %}
+            </div>
+        {% endif %}
+
+        {% if forloop.last and not forloop.first %}
```

Comment:
shouldn't `foorloop.last` be enough? Can `forloop.first` and `forloop.last` be both true at the same time?